### PR TITLE
Adds options form, identifiable object forms and grid hooks docs

### PR DESCRIPTION
**Pull request by tomas862**

### CODE REVIEW

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 1]
```diff
@@ -2135,4 +2135,152 @@ action<KpiIdentifier>KpiRowModifier
     ),
     ```
 
-{{% /funcdef %}}
+action&lt;HookName>Form
+:
+<!--     todo: options form available since ?-->
```
Comment (tomas862) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md]: @matks do you know when the options form hooks were introduced - in which PrestaShop version?

Comment (matks) [reply src/content/1.7/modules/concepts/hooks/list-of-hooks.md]: 1.7.4 https://github.com/PrestaShop/PrestaShop/pull/8279

**Comment by tomas862** [comment]
@matks done

**Comment by matks** [comment]
Thank you @tomas862

**Pull request merged**